src/containerapp/azext_containerapp/_utils.py

```diff
@@ -556,6 +556,24 @@ def _new_tiny_guid():
     return ''.join(random.choices(string.ascii_letters + string.digits, k=4))
 
 
+#  Generate a random volume name using same method as log analytics workspace
+def _generate_secret_volume_name():
+    import re
+    prefix = "secret-volume"
+    # volume name must be lowercase
+    suffix = _new_tiny_guid().lower()
+    maxLength = 40
```

Comment:
Do we have similar check in CP ?

Reply:
yes we do https://msazure.visualstudio.com/One/_git/AAPT-Antares-Containerapps?path=/src/Microsoft.ContainerApps/Microsoft.ContainerApps.Common/Validators/ContainerAppsApiValidator.cs&version=GBmain&line=1405&lineEnd=1406&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
MaxPropertyValueLength = 263;

Any reasons for setting this to 40 here ?

https://msazure.visualstudio.com/One/_git/AAPT-Antares-Containerapps?path=/src/Microsoft.ContainerApps/Microsoft.ContainerApps.Common/Validators/ContainerAppsApiValidator.cs&version=GBmain&line=26&lineEnd=26&lineStartColumn=27&lineEndColumn=49&lineStyle=plain&_a=contents

Reply:
okay I see we have similar limits for workspace name and others.